mayastor/src/nvmf_target.rs

```diff
@@ -245,6 +246,37 @@ impl Iterator for SubsystemIter {
         }
     }
 }
+///
+/// Some options can be passed into each target that gets created.
+///
+/// Currently the options are limited to the name of the target to be created
+/// and the max number of subsystems this target supports. We set this number
+/// equal to the number of pods that can get scheduled on a node which is, by
+/// default 110.
+pub(crate) struct TargetOpts {
+    inner: spdk_nvmf_target_opts,
+}
+
+impl TargetOpts {
+    fn new(name: &str, max_subsystems: u32) -> Self {
+        let mut opts = spdk_nvmf_target_opts::default();
+
+        unsafe {
+            std::ptr::copy_nonoverlapping(
+                name.as_ptr() as *const _ as *mut libc::c_void,
+                &mut opts.name[0] as *const _ as *mut libc::c_void,
+                256,
```

Comment:
just a nit. using spdk_sys::NVMF_TGT_NAME_MAX_LENGTH would be safer here.

Reply:
I've added a new issue for this because for some odd reason its not there, I think its due to the white labelling but, things used should be automatically white listed as well.